DESCRIPTION

```diff
@@ -23,29 +23,41 @@ License: Apache License 2.0 | file LICENSE
 URL: https://github.com/insightsengineering/rtables,
     https://insightsengineering.github.io/rtables/
 BugReports: https://github.com/insightsengineering/rtables/issues
-Depends: 
+Depends:
     formatters (>= 0.5.0),
-    magrittr,
+    magrittr (>= 1.5),
     methods,
     R (>= 2.10)
 Imports:
     grid,
-    htmltools,
+    htmltools (>= 0.5.4),
     stats
 Suggests:
-    dplyr,
-    flextable,
-    knitr,
-    officer,
+    dplyr (>= 1.0.0),
+    flextable (>= 0.5.10),
+    knitr (>= 1.42),
+    officer (>= 0.3.10),
     r2rtf,
-    rmarkdown,
-    survival,
-    testthat,
-    tibble,
+    survival (>= 2.43-1),
+    testthat (>= 3.0.4),
+    tibble (>= 3.1.0),
     tidyr,
-    xml2
-VignetteBuilder: 
+    xml2 (>= 1.1.0)
+VignetteBuilder:
     knitr
+Remotes:
+    insightsengineering/formatters@*release,
```

Comment:
I don't see a reason to add this, we should talka bout it before merging this in.

Generally I'm against specifying dependencies should be installed from specific places, as it is fundamentally the wrong approach and the control of that belongs in the hands of the end user (and changing where deps come from should NOT involve editing the package itself, which it does in the ill-advised remotes paradigm)

Reply:
changed this into `Config/Needs/verdepcheck` field of DESCRIPTION file